components/EntityAttributeValue.js

```diff
@@ -42,6 +42,10 @@ const formatImage = (key) => {
       case 'float':
         //TODO: Find a more accurate representation of float
         return Number(item.value_double);
+      //added for datetime
+      case 'date-time':
+        return item.value_string;;
```

Comment:
Please see my previous comments. 

```suggestion
        case 'datetime':
        return item.value_datetime;
```